pkg/applicationserver/config.go

```diff
@@ -148,9 +148,6 @@ func (c PubSubConfig) toProto() *ttnpb.AsConfiguration_PubSub {
 	if status, ok := c.Providers["nats"]; ok {
 		providers.Nats = toStatus(status)
 	}
-	if status, ok := c.Providers["awsiot"]; ok {
```

Comment:
This is already the case in Enterprise. See https://github.com/TheThingsIndustries/lorawan-stack/blob/9912dddafff50e27882e93711ebb3007c22e1047/pkg/applicationserver/config.go#L150-L159